composer.json

```diff
@@ -21,8 +21,7 @@
         "php": "^7.1",
         "behat/behat": "^3.0",
         "behat/mink-extension": "^2.3",
-        "behat/symfony2-extension": "^2.1",
-        "sonata-project/user-bundle": "^4.0"
+        "behat/symfony2-extension": "^2.1"
```

Comment:
it would be nice to allow `behat/symfony2-extension`  and `friends-of-behat/symfony-extension`